pandas/tests/series/test_timezones.py

```diff
@@ -379,3 +379,11 @@ def test_constructor_data_aware_dtype_naive(self, tz_aware_fixture):
         result = Series([Timestamp("2019", tz=tz)], dtype="datetime64[ns]")
         expected = Series([Timestamp("2019")])
         tm.assert_series_equal(result, expected)
+
+    def test_numpy_reduction_with_tz_aware_dtype(self, tz_aware_fixture):
+        # GH 15552
+        tz = tz_aware_fixture
+        arg = to_datetime(["2019"]).tz_localize(tz)
+        expected = Series(arg)
+        result = np.minimum(expected, expected)
+        tm.assert_series_equal(result, expected)
```

Comment:
Might belong in tests.reductions?  Especially if it can be extended to the other box classes